.circleci/config.yml

```diff
@@ -325,38 +325,32 @@ jobs:
             fi
       - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run:
-          name: 'Generate Version Number'
-          command: echo "export VERSION=$(npm run --silent get-next-version | xargs)" >> $BASH_ENV
-      - run: echo ${VERSION}
+          name: 'Store current version number'
+          command: echo "export PREVIOUS_VERSION=$(npm run --silent tooling -- version current | xargs)" >> $BASH_ENV
       - run:
-          name: 'Build Packages'
-          command: npm run build
-      - run:
-          name: 'Build UMD script'
-          command: npm run build:script
-      - run:
-          name: 'Bump all package version'
-          command: npm run tooling -- version set ${VERSION} --all
-      - run:
-          name: 'Commit bumped version'
-          command: |
-            git add packages/node_modules/*/package.json packages/node_modules/@ciscospark/*/README.md packages/node_modules/@webex/*/package.json packages/node_modules/webex/umd/*.js
-            git commit -m "chore(release): v${VERSION} [skip ci]"
+          name: 'Run standard-version'
+          command: npm run release
       - run:
           name: 'Push to upstream/master'
```

Comment:
Should this move along with the tag push?

Reply:
Nope, this should be pushed at least, kinda weird to publish to npm first and then github
It's probably easier to publish to npm locally then deal with npm

Reply:
That being said if a failure happens during npm publish, then the git tag doesnt get pushed which is probably another problem, so i'm more inclined to move git tag back up to here

Reply:
I think is ok for nothing to be pushed to git if the publishing failed. The commits that will get pushed are on all version/publishing changes right?
So if publishing a new version failed we should probably not push those things into git anyway. This way we can re-run the publishing workflow as if the last stuff was never versioned-bumped.

Reply:
This is probably a bigger problem if some of the packages get published and some don't

Reply:
good point, there are multiple packages to publish. What's the flow if something fails?

Reply:
I think i'd be fine if the github release and tag are committed before npm
If a failure happens during npm publish we (i.e. npm admin adam) will have to manually push it up

Reply:
If it ever becomes a problem we can swap to push + commit after npm publish
But i'd rather keep the setup the same~ish and other tools suggest push+tag->publish

Reply:
@adamweeks, for the record so you know what @taymoork2 is signing you up for 😆

Reply:
He already knows, even I dont have access to publish on npm, I think him and Dave are the only ones who can atm